java/client/src/main/java/glide/api/models/commands/stream/StreamGroupOptions.java

```diff
@@ -0,0 +1,53 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.stream;
+
+import glide.api.commands.StreamBaseCommands;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+
+/**
+ * Optional arguments for {@link StreamBaseCommands#xgroupCreate(String, String, String,
+ * StreamGroupOptions)}
+ *
+ * @see <a href="https://valkey.io/commands/xgroup-create/">valkey.io</a>
+ */
+@Builder
```

Comment:
I think user API with 3 constructors or factory methods will be easier

Reply:
Are you suggesting: 
```
xgroupCreate(String key, String groupname, String id)
xgroupCreate(String key, String groupname, String id, boolean makeStream)
xgroupCreate(String key, String groupname, String id, String entriesRead)
xgroupCreate(String key, String groupname, String id, boolean makeStream, String entriesRead)
```

Reply:
```java
new StreamGroupOptions(bool)
new StreamGroupOptions(bool, String)
new StreamGroupOptions(String)
```

Reply:
ok...